Text/PrettyPrint/Leijen/Text.hs

```diff
@@ -1005,6 +1018,9 @@ c `consB` b = B.singleton c `mappend` b
 displayT :: SimpleDoc -> Text
 displayT = B.toLazyText . displayB
 
+displayTStrict :: SimpleDoc -> TS.Text
+displayTStrict = T.toStrict . displayT
```

Comment:
Hmm, I wasn't able to find one. Where is it? I think there might be one in `text-show` which would be a new dependency.